boards/v2/arm/mps3_an547/board.yml

```diff
@@ -0,0 +1,7 @@
+board:
+  name: mps3
```

Comment:
typo? this is not the board name afaik.

Reply:
I would say it's the the board, ref: https://www.arm.com/products/development-tools/development-boards/mps3

and then you build for the an547 SoC (image) by using `-DBOARD=mps3/an547` as identifier.

Reply:
but should probably rename the folder itself to just be `mps3`.